fpm/src/fpm/manifest/package.f90

```diff
@@ -229,6 +240,11 @@ subroutine info(self, unit, verbosity)
             write(unit, fmt) "- name", self%name
         end if
 
+        if (allocated(self%build_config)) then
+            write(unit, fmt) "- build configuration", ""
```

Comment:
This header can be written by the `build_config` object, for the package itself it is just opaque.
```suggestion
```